http-sink/src/main/java/io/streamzi/router/sink/eventbus/HttpEventPublisher.java

```diff
@@ -20,25 +32,37 @@
 
     @Override
     public void startStromBrauVerticle(final ConfigRetriever retriever) {
-        logger.info("Starting Event Router");
-
+        logger.info("Kafka to HTTP Bridge");
         client = vertx.createHttpClient();
-
         retriever.rxGetConfig().subscribe(myconf -> registerHandler(myconf));
     }
 
     private void registerHandler(final JsonObject myconf) {
 
-                eventBus.consumer(CE_ADDRESS)
-                .toFlowable()
-                .subscribe(message -> {
 
-                    final CloudEvent ce = Json.decodeValue(message.body().toString(), CloudEventImpl.class);
-                    logger.info("Publishing event to HTTP (" + ce.getEventType() +")");
+        final Map consumerConfig = new Properties();
+        consumerConfig.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, myconf.getString("MY_CLUSTER_KAFKA_SERVICE_HOST") + ":" + myconf.getInteger("MY_CLUSTER_KAFKA_SERVICE_PORT").toString());
+
+        // if STREAMZI_GROUP is not defined, we use a UUID, to guarantee unique consumer group, per EVENT_TYPE
+        final String streamziGroup = myconf.getString("STREAMZI_GROUP") != null ? myconf.getString("STREAMZI_GROUP") : UUID.randomUUID().toString();
+        consumerConfig.put(ConsumerConfig.GROUP_ID_CONFIG, "io.streamzi.kafka.cloudevent." + myconf.getString("STREAMZI_EVENT_TYPE") + streamziGroup);
```

Comment:
Do you want a "." between the EVENT_TYPE and streamziGroup?

Reply:
ha! yeah - good catch